### ISSUE TITLE
Add capability to capture more granular sightings counts

### ISSUE THREAD

**Issue by johnwunder** [STIXProject/schemas#240]
It's not possible to capture sightings counts at more granular levels than the indicator itself. It might be useful, however, to say that a particular sighting was seen 253 times by organization A and 213 times by Org B and 0 times by org C.

**Comment by jordan2175** [comment]
Yes, and we need to make sure this easily allows for the facebook +1 feature of hey I have seen this too.
